pkgs/development/tools/analysis/qcachegrind/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchurl, bash, cmake, qt57, graphviz,  qmakeHook, makeQtWrapper, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  name = "kcachegrind-${version}";
+  version = "16.12.2";
+
+  src = fetchurl {
+    url = "http://download.kde.org/stable/applications/${version}/src/${name}.tar.xz";
```

Comment:
I am not sure if it is possible at all to provide linux+macos outputs through the `kdeWrapper`. I think it is best for bost OSes to have the two packages separate, since the `CMakelists.txt` for linux depend on `find_library(kde4)` which will never work on MacOS. Thus, a simple separate qcachegrind package for non-linux users will be the alternative.